src/core/src/op/abs.cpp

```diff
@@ -69,6 +91,27 @@ bool Abs::has_evaluate() const {
         return false;
     }
 }
+
+bool Abs::evaluate_lower(TensorVector& output_values) const {
+    return ov::op::abs::evaluate_bound(this, get_input_tensor(0), output_values, true);
+}
+
+bool Abs::evaluate_upper(TensorVector& output_values) const {
+    return ov::op::abs::evaluate_bound(this, get_input_tensor(0), output_values, false);
+}
+
+bool Abs::evaluate_symbol(TensorSymbolVector& output_symbols) const {
+    const auto& non_negative = ov::util::to_vector<bool>(ov::util::ge(get_input_tensor(0).get_lower_value(), 0));
+    const auto& input_symbols = get_input_tensor(0).get_value_symbol();
+    if (non_negative.empty() || input_symbols.size() != non_negative.size())
+        return false;
+    output_symbols.resize(1);
+    output_symbols[0].resize(non_negative.size(), nullptr);
+    for (size_t i = 0; i < non_negative.size(); ++i)
+        if (non_negative[i])
+            output_symbols[0][i] = input_symbols[i];
+    return true;
```

Comment:
```suggestion
    auto non_negative_ptr = non_negative.data<char>();
    for (size_t i = 0; i < non_negative.size(); ++i)
        if (non_negative_ptr[i])
            output_symbols[0][i] = input_symbols[i];
    return true;
```
as `non_negative` is known type cast to vector is not required.